ngraph_bridge/ngraph_executor.cc

```diff
@@ -308,7 +308,8 @@ NGraphExecutor::CreateCallback(const std::string signature,
         "tf_function_" + m_node_name + "_" + to_string(rank_id) + ".json",
         serialized_ng_func);
     if (status != Status::OK()) {
-      return std::make_pair(status, std::make_pair(ng_exec, ng_function));
+      return std::make_pair(status,
```

Comment:
Isn't this the distributed build fix change that went in already?

Reply:
that went in the master, this is on r0.21 to create a tag

Reply:
May be you want to cherry-pick it separately so that tagging can be a stand-alone PR

Reply:
Its fine. Not a big change